app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
Again, we want to use `find_by` rather than `where().first` - make sure you know the difference.